repositories/src/main/scala/cromwell/pipeline/datastorage/dto/formatters/AuthFormatters.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.pipeline.datastorage.dto.formatters
+
+import play.api.libs.json.{ Json, OFormat }
+
+object AuthFormatters {
+
+  final case class AuthResponse(accessToken: String, refreshToken: String, accessTokenExpiration: Long)
```

Comment:
This class should not be moved here. `AuthFormatters` object must only contain formatters for the classes, not the classes themself.
Same for all other classes (including `ProjectAdditionRequest`, `Version` etc)

Reply:
You suggest storing companion-objects separate from case classes?

Reply:
No, I don't.
Formatters doesn't have to be inside companion object. Moreover, your main goal is to get them out of these objects. 
`Currently there are implicit json formatters in different classes and objects. It would be better to keep them in one place.`
It should be something like this 
```
object AuthFormatters {
  implicit val authResponseFormat: OFormat[AuthResponse] = Json.format[AuthResponse]
  implicit val signInRequestFormat: OFormat[SignInRequest] = Json.format[SignInRequest]
}
```

Reply:
I discus this question with another mentors on daily. Thank you.

Reply:
I have a problem with that, because in this case I have a compilation error "could not find implicit value for parameter" in classes, when I use case classes.

Reply:
fixed